src/igvfd/schemas/changelogs/analysis_step.md

```diff
@@ -2,6 +2,7 @@
 
 ### Minor changes since schema version 5
 
+* Extend `analysis_step_types` enum list to include `barcode counting`, `umi quantification`, `cell to feature barcode mapping`, `demultiplexing`, `differential expression analysis`.
```

Comment:
Analysis_step changelog is updated, but the actual .json schema is not change.

Reply:
oh i forgot to save. I will push with the changes to json schema page